packages/dashboard/src/components/queryEditor/advancedSearch/advancedSearch.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import SpaceBetween from '@cloudscape-design/components/space-between';
+import Button from '@cloudscape-design/components/button';
+import FormField from '@cloudscape-design/components/form-field';
+import Input from '@cloudscape-design/components/input';
+import { useState, useMemo, useCallback } from 'react';
+import { ExecuteQueryCommandOutput } from '@aws-sdk/client-iottwinmaker';
+import { createKnowledgeGraphQueryClient } from '@iot-app-kit/react-components/src/components/knowledge-graph/KnowledgeGraphQueries';
+import { TwinMakerKGQueryDataModule } from '@iot-app-kit/source-iottwinmaker';
+import { AdvancedResultsTable } from './advancedResultsTable';
+import { Property } from './model/property';
+
+export interface AdvancedSearchProps {
+  kGDatamodule: TwinMakerKGQueryDataModule;
+  onSelect: (properties: Property[]) => void;
+}
+
+/*
+    The return types are incorrect for Knowledge Graph
+    when selecting an edge instead of a node
+    so we can't properly type here.
+
+    When SiteWise sync is turned on with Knowledge Graph
+    it adds a definition field for the propertyId and displayName
+    and adds propertyName as a field.
+  */
```

Comment:
Why is the formatting off on this comment?

Reply:
I thought the linting would fix it, updated it in a new commit I pushed.